### PR TITLE
Quick Facts sheet update for French

### PR DESCRIPTION
**Pull request by el00ruobuob**
This is the French part of #100.  
It does include:  
* translation of changes
* a few minor correction to french wording
* suggestions i made in #99

### CODE REVIEW

**Comment by erciccione** [comment]
@assylias You know what i'm about to ask :P

File: monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md [hunk 1]
```diff
@@ -26,67 +26,68 @@ En tant que cryptomonnaie décentralisée, Monero est sécurisé par un grand r
...
+peuvent décider précisément quelles informations ils révèlent et à qui.
 
-#### CONFIDENTIALITÉ
+#### RÉSISTANCE À LA CENSURE
 
-Monero utilise une cryptographie sophistiquée avec les _[signatures de cercle](https://getmonero.org/fr/resources/moneropedia/ringsignatures.html)_, les _[transactions confidentielles de cercle](https://getmonero.org/fr/resources/moneropedia/ringCT.html)_, et les _[adresses furtives](https://getmonero.org/fr/resources/moneropedia/stealthaddress.html)_ pour masquer les origines, les montants et les destinations de toutes les transactions.
+Grâce au caractéristiques de confidentialité de Monero, aucune adresse ou utilisateur spécifique ne peut être mis sur liste noir par les mineurs ou par un acteur économique. De ce fait, les utilisateurs sont libérés de la censure et des _[contrôles de capitaux](https://fr.wikipedia.org/wiki/Contr%C3%B4le_des_capitaux)_.
```
Comment (assylias) [inline monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md]: noir -> noire

File: monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md [hunk 2]
```diff
@@ -26,67 +26,68 @@ En tant que cryptomonnaie décentralisée, Monero est sécurisé par un grand r
...
 
 #### FONGIBILITÉ
 
-Monero est _[fongible](https://getmonero.org/fr/resources/moneropedia/fungibility.html)_ car il est confidentiel par défaut. Cela signifie qu'un Monero sera toujours égal à un autre. Les unités de Monero ne peuvent pas être discriminées par les commerçants ou les bourses à cause de l'origine ou l'historique de vos pièces.
+Monero est _[fongible](https://getmonero.org/fr/resources/moneropedia/fungibility.html)_ car il est confidentiel par défaut. Il n'y a pas d'historique visible associé à chaque pièces particulières. cela signifie que les utilisateurs et les sociétés n'ont pas à se soucier d'être accusé d'accepter de l'argent sale, et qu'un Monero sera toujours égal à un autre.
```
Comment (assylias) [inline monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md]: chaque pièces particulières -> chaque pièce particulière  
cela -> Cela

File: monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md [hunk 3]
```diff
@@ -26,67 +26,68 @@ En tant que cryptomonnaie décentralisée, Monero est sécurisé par un grand r
...
 
 ### POINTS CLEFS DIFFÉRENCIATEURS
 
 **Monero utilise la base de code de CryptoNote :** Elle est fondamentalement différente des bases de codes utilisées par Bitcoin ou Ethereum et les nombreuses autres cryptomonnaies dérivées de celles-ci. Elle est connue pour ses nombreuses améliorations de la confidentialité.
 
-**Confidentialité obligatoire, transparence en option :** La cryptographie à l'état de l'art de Monero masque tous les niveaux d'une transaction : les informations de l'émetteur, du destinataire, ou la transaction elle-même. Si un utilisateur préfère la transparence, il peut la choisir en partageant un portefeuille d'audit qui révèle ses entrées.
+**Confidentialité obligatoire, transparence en option :** Monero est confidentiel à tous les niveaux d'une transaction : les informations de l'émetteur, du destinataire, ou la transaction elle-même. Un utilisateur à la possibilité de créer et de partager un portefeuille d'audit qui révèle ses entrées, ou d'utiliser des clefs de visualisation pour révéler des transactions spécifiques.
```
Comment (assylias) [inline monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md]: à la possibilité -> a (no accent)

File: monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md [hunk 4]
```diff
@@ -26,67 +26,68 @@ En tant que cryptomonnaie décentralisée, Monero est sécurisé par un grand r
...
 
-**Laboratoire de Recherche Monero :** Monero n'est pas uniquement voué à créer une cryptomonnaie fongible, mais aussi à continuer la recherche dans le domaine de la confidentialité financière dans la mesure où cela fait appel aux cryptomonnaies d'une manière générale. Dans ce but, des chercheurs en en mathématiques et physique numérique ont publiés **[livres blancs](https://lab.getmonero.org/)** et travaillent sur bien d'autres **[domaines de recherche](https://getmonero.org/fr/design-goals/)** sur lesquels ils travaillent.
+**Laboratoire de Recherche Monero :** Monero n'est pas uniquement voué à créer une cryptomonnaie fongible, mais aussi à continuer la recherche dans le domaine de la confidentialité financière dans la mesure où cela fait appel aux cryptomonnaies d'une manière générale. Dans ce but, des chercheurs universitaire dans les domaines des mathématiques, physique, sécurité et calculs de chaîne de blocs ont publiés des **[articles de recherche](https://lab.getmonero.org/)** et travaillent sur bien d'autres **[objectifs](https://getmonero.org/fr/design-goals/)**.
```
Comment (assylias) [inline monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md]: des chercheurs universitaire -> universitaires (with an s)

File: monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md (hunk 1 above)
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md]: ok

File: monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md (hunk 2 above)
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md]: ok

File: monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md (hunk 3 above)
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md]: ok

File: monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md (hunk 4 above)
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr.md]: ok

**Comment by erciccione** [comment]
Thank you @el00ruobuob @assylias . You are both amazing! :)

**Pull request merged**